src/styles/catalog_section.scss

```diff
@@ -0,0 +1,9 @@
+.catalog__section {
+  padding-top: 50px;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
+  grid-template-rows: 1fr 1fr;
```

Comment:
You don't need "grid-template-rows: 1fr 1fr;" as it could be more cards and rows